bin/registerServerTopology.php

```diff
@@ -0,0 +1,283 @@
+<?php
+
+/*
+ * Copyright 2005 - 2020 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+require_once('registerServerTopology-func.php');
+
+/************************ */
+/*     DATA RESOLVING
+/************************ */
+
+/**
+ * Get script params
+ */
+$opt = getopt('u:t:h:n:', ["help::", "dns:", "insecure::"]);
+const SERVER_TYPES = ["poller", "remote", "map", "mbi"];
+
+/**
+ * Format the --help message
+ */
+$helpMessage = <<<'EOD'
+
+Global Options:
```

Comment:
How do you manage automated / silenced deployement using a payload ?
eg : a file with mandatory parameters ?